static/src/scss/resources/_resources.scss

```diff
@@ -453,12 +461,9 @@ footer.resource-detail__stats {
 }
 
 .resource-abstract {
-    font-size: 1.125rem;
+    font-size: 0.85rem;
     margin-top: 0;
-
-    .private & {
```

Comment:
This probably needs to stay in as the key icon lives in the top right of the tile